meta-resin-common/recipes-connectivity/networkmanager/resin-files/99dhcp_ntp

```diff
@@ -28,7 +28,7 @@ case "$action" in
 		for ntp in ${ntp_connection}; do
 			# Only delete the server if it isn't being added by some other interface
 			if ! grep -q "$ntp" $added_servers_file; then
-					/usr/bin/chronyc delete $ntp
+					/usr/bin/chronyc delete $ntp || true
```

Comment:
Some strangeness with the alignment here.

Reply:
fixed